README.md

```diff
@@ -244,6 +244,19 @@ In the example the current list `.sortable` allows items within it to be sorted
 
 If you want to be able to move items between to sortables, the `acceptFrom` option must be present on both of them.
 
+Can also accept a callback function with `destinationSortable` and `draggedElement` as paramaters, This function should return `true` or `false`:
```

Comment:
Can you please change this to:

You can also provide a function for the `acceptFrom` option. The function will receive two parameters, `destinationSortable` and `draggedElement`. Your function must return a boolean value (`true` or `false`).